src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -530,19 +456,13 @@ private bool ValidateNonce(PublicEntry entry, ExecutionEnvironment env, ITxTrace
         /// <exception cref="TransactionCollisionException">Thrown when the address is already in use</exception>
         private void PrepareContractAccount(Address recipient)
         {
-            if (!_stateProvider.AccountExists(recipient))
-            {
-                return;
-            }
-            
+            if (!_stateProvider.AccountExists(recipient)) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)